docs/.sphinx/requirements.txt

```diff
@@ -0,0 +1,269 @@
+click==8.1.3
+    # via
+    #   jupyter-cache
+    #   sphinx-external-toc
+comm==0.1.2
+    # via ipykernel
+debugpy==1.6.6
+    # via ipykernel
+decorator==5.1.1
+    # via ipython
+deprecated==1.2.13
+    # via pygithub
+docutils==0.16
+    # via
+    #   breathe
+    #   myst-parser
+    #   pydata-sphinx-theme
+    #   rocm-docs-core
+    #   sphinx
+executing==1.2.0
+    # via stack-data
+fastjsonschema==2.16.2
+    # via nbformat
+gitdb==4.0.10
+    # via gitpython
+gitpython==3.1.30
+    # via rocm-docs-core
+greenlet==2.0.2
+    # via sqlalchemy
+idna==3.4
+    # via requests
+imagesize==1.4.1
+    # via sphinx
+importlib-metadata==6.0.0
+    # via
+    #   jupyter-cache
+    #   myst-nb
+ipykernel==6.21.2
+    # via myst-nb
+ipython==8.10.0
+    # via
+    #   ipykernel
+    #   myst-nb
+jedi==0.18.2
+    # via ipython
+jinja2==3.1.2
+    # via
+    #   myst-parser
+    #   sphinx
+jsonschema==4.17.3
+    # via nbformat
+jupyter-cache==0.5.0
+    # via myst-nb
+jupyter-client==8.0.3
+    # via
+    #   ipykernel
+    #   nbclient
+jupyter-core==5.2.0
+    # via
+    #   ipykernel
+    #   jupyter-client
+    #   nbformat
+linkify-it-py==1.0.3
+    # via myst-parser
+markdown-it-py==2.2.0
+    # via
+    #   mdit-py-plugins
+    #   myst-parser
+markupsafe==2.1.2
+    # via jinja2
+matplotlib-inline==0.1.6
+    # via
+    #   ipykernel
+    #   ipython
+mdit-py-plugins==0.3.4
+    # via myst-parser
+mdurl==0.1.2
+    # via markdown-it-py
+myst-nb==0.17.1
+    # via rocm-docs-core
+myst-parser[linkify]==0.18.1
+    # via
+    #   myst-nb
+    #   rocm-docs-core
+nbclient==0.5.13
+    # via
+    #   jupyter-cache
+    #   myst-nb
+nbformat==5.7.3
+    # via
+    #   jupyter-cache
+    #   myst-nb
+    #   nbclient
+nest-asyncio==1.5.6
+    # via
+    #   ipykernel
+    #   nbclient
+packaging==23.0
+    # via
+    #   ipykernel
+    #   pydata-sphinx-theme
+    #   sphinx
+parso==0.8.3
+    # via jedi
+pexpect==4.8.0
+    # via ipython
+pickleshare==0.7.5
+    # via ipython
+platformdirs==3.0.0
+    # via jupyter-core
+prompt-toolkit==3.0.37
+    # via ipython
+psutil==5.9.4
+    # via ipykernel
+ptyprocess==0.7.0
+    # via pexpect
+pure-eval==0.2.2
+    # via stack-data
+pycparser==2.21
+    # via cffi
+pydata-sphinx-theme==0.13.0rc5
+    # via sphinx-book-theme
+pygithub==1.57
+    # via rocm-docs-core
+pygments==2.14.0
+    # via
+    #   accessible-pygments
+    #   ipython
+    #   pydata-sphinx-theme
+    #   sphinx
+pyjwt==2.6.0
+    # via pygithub
+pynacl==1.5.0
+    # via pygithub
+pyrsistent==0.19.3
+    # via jsonschema
+python-dateutil==2.8.2
+    # via jupyter-client
+pytz==2022.7.1
+    # via babel
+pyyaml==6.0
+    # via
+    #   jupyter-cache
+    #   myst-nb
+    #   myst-parser
+    #   sphinx-external-toc
+pyzmq==25.0.0
+    # via
+    #   ipykernel
+    #   jupyter-client
+requests==2.28.1
+    # via
+    #   pygithub
+    #   sphinx
+rocm-docs-core @ git+https://github.com/RadeonOpenCompute/rocm-docs-core.git
```

Comment:
A specific commit hash should be used here.

Reply:
Sam and I are preparing a bot account to make PRs to update this hash. We left it without a commit hash due to hourly changes we are making to rocm-docs-core. The intention is to pin it but in the interest of rapid development, we were a little lazy. If you want, we can pin it now or pin it for the rest of ROCm project simultaneosly via autogenerate PRs from the bot account.

Reply:
Done in [pin rocm-docs-core](https://github.com/ROCmSoftwarePlatform/MIOpen/pull/2053/commits/3316da521c40ed56726d3e7106218db852d7ed97)